ui/src/app/components/admin/audit-page-component.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
+import { Redirect } from 'react-router';
 import * as fp from 'lodash/fp';
 
 import { AuditAction, AuditLogEntry } from 'generated/fetch';
 
 import { AuditActionCardListView } from 'app/components/admin/audit-card-list-view';
-import { Navigate } from 'app/components/app-router';
```

Comment:
There is a react-router 6 component Navigate - removing our custom component with the same name (which we don't actually need) to avoid confusion later.